### PR TITLE
changing default opponent to be Trigram Tracy

### PR DESCRIPTION
**Pull request by azax1**
@miorel this apparently makes the select "uncontrolled". I'd like for you to check that I invoked the idioms correctly

### CODE REVIEW

File: lexicant.html [hunk 1]
```diff
@@ -613,18 +613,21 @@ <h1>Lexicant game</h1>
...
+        <div>
+          <span>
             Choose your opponent...
-          </option>
-          <option value="Random">Random Randi</option>
-          <option value="Careful">Careful Carter</option>
-          <option value="Perfect">Perfect Percy</option>
-          <option value="Trigram">Trigram Tracy</option>
-        </select>
+            <br />
```
Comment (miorel) [inline lexicant.html:619]: Typically we want the label of a component to be semantically marked in the HTML, I would suggest using a `<label>` here with its `htmlFor` attribute set to the `<select>` below. See for example: https://react.dev/reference/react-dom/components/select#providing-a-label-for-a-select-box

File: lexicant.html [hunk 2]
```diff
@@ -613,18 +613,21 @@ <h1>Lexicant game</h1>
...
+          </span>
+          <select
+            defaultValue={"Trigram"}
+            onChange={(ev) => setOpponent(ev.target.value)}
```
Comment (miorel) [inline lexicant.html:623]: This does indeed make it uncontrolled. Typically for uncontrolled components one doesn't provide an onChange either, instead reading the value of the element when something else happens (e.g. some form is submitted).

I think the way you're doing this is fine, but given that you've moved the label outside anyway, is it possible to use a state like before, with "Trigram" as its initial value?

**Review by miorel — APPROVED** [review]
Thanks for doing this! I think this is commitable even as-is, but I added a couple of comments for your consideration.

**Comment by azax1** [comment]
Closing w/o merge because the essential functional change was made in d2c42ed254c2e while leaving the select controlled

**Pull request closed without merging**